Library/Homebrew/tap.rb

```diff
@@ -284,8 +284,8 @@ def install(options = {})
 
     link_completions_and_manpages
 
-    formula_count = formula_files.size
-    puts "Tapped #{Formatter.pluralize(formula_count, "formula")} (#{path.abv})" unless quiet
+    formatted_contents = Formatter.comma_and(*contents)&.prepend(" ")
```

Comment:
Thoughts about the `prepend` being part of `comma_and`?

Reply:
I don't think it's worth it.

Reply:
My thinking was it's currently an accessory function where the only use is with the `prepend`?

Reply:
Still, I think it‘s not worth it:

```ruby
Formatter.comma_and(*contents)&.prepend(" ")
 ```

vs.

```ruby
Formatter.comma_and(*contents, prepend: " ")
 ```
and a more complex `comma_and`.